numba/core/untyped_passes.py

```diff
@@ -1054,7 +1054,10 @@ def get_call_args(init_arg, want):
                                                     stmt.value.value)
                                         if dfn is None:
                                             continue
-                                        args = getattr(dfn, 'args', False)
+                                        try:
+                                            args = getattr(dfn, 'args', False)
+                                        except KeyError:
```

Comment:
The `getattr` should not have failed with raising a `KeyError`. I think there's a bug in `class Expr`. It's `__getattr__` should do:

```python

    def __getattr__(self, name):
        if name.startswith('_'):
            return Inst.__getattr__(self, name)
        try:
            return self._kws[name]
        except KeyError:
            raise AttributeError(name)
```